src/main/java/Duke.java

```diff
@@ -9,15 +9,18 @@
 
     /** Stores all task added by the user */
     public static TaskManager taskList = new TaskManager();
+
     /** Gets a new instance of UI class to interactive with user */
     public static UI UIInteract = new UI(USERNAME);
```

Comment:
I'm not sure about this since UI is an acronym, but would it better to use it as camel case and label it as uiInteract?